policy/README.md

```diff
@@ -0,0 +1,33 @@
+# CNCF Security SIG Policy Team Introduction
+
+## Background
+
+When SAFE WG was originally proposed, developers from Kubernetes Policy WG
+actively participated the discussions and then proposed CNCF level Policy WG was
+[merged](../policy-wg-merging.md) with SAFE WG to form the current CNCF Security
+SIG, in alignment with the SIG [charter](../governance/charter.md).
+
+## Policy team
+
+The Policy Team in CNCF Security SIG consists of the same set of developers that
+forms the [Kubernetes Policy WG](https://github.com/kubernetes/community/tree/master/wg-policy). 
+The only difference is the topic coverage (whether it is at Kubernetes level or
+CNCF level).
+
+There are now two active co-leads for Policy Team @hannibalhuang and @ericavonb.
```

Comment:
There are now two active co-leads for the Policy Team, @hannibalhuang and @ericavonb.

Reply:
done

Reply:
In a diff PR one of the SIG members gave me feedback that we should keep all the names of people filling particular roles separate from the description of the roles and the governance of how those roles are decided.  So, I added lead roles to the root README, including policy team leaders here:  https://github.com/cncf/sig-security#policy-team

I think it is better that this doc talk about the roles in general, then link to where individuals are named (to make the info easier to find, reference, update).

Reply:
> In a diff PR one of the SIG members gave me feedback that we should keep all the names of people filling particular roles separate from the description of the roles and the governance of how those roles are decided. So, I added lead roles to the root README, including policy team leaders here: https://github.com/cncf/sig-security#policy-team
> 
> I think it is better that this doc talk about the roles in general, then link to where individuals are named (to make the info easier to find, reference, update).

Done